examples/hotrod/services/frontend/server.go

```diff
@@ -76,7 +76,7 @@ func (s *Server) Run() error {
 }
 
 func (s *Server) createServeMux() http.Handler {
-	mux := tracing.NewServeMux(true, s.tracer, s.logger)
+	mux := tracing.NewServeMux(s.tracer, s.logger)
```

Comment:
you could've kept the boolean parameter to minimize the changes and PR each server separately. Then at the end we can do a round of clean-up. As it stands now, you have a PR with a dozen different things going on.